src/usr/local/pkg/miniupnpd.xml

```diff
@@ -161,7 +160,7 @@
 				Example public STUN servers:
 				<ul>
 					<li>stun.counterpath.com</li>
-					<li>stun.cloudflare.com</li>
+					<li>stun.nextcloud.com</li>
```

Comment:
> it was recently discovered that the daemon needs certain features and doesn't check for their support, leading to incorrect STUN CGNAT check results and disabling the service.

Is there a link to this discussion / more details?

Reply:
Thanks. This seems fine to me given miniupnpd has updated their default:
https://github.com/miniupnp/miniupnp/commit/3f0316584ab71876e89f401ee193a63c79bcf00e